lib/src/internal/editor.dart

```diff
@@ -9,16 +9,28 @@ import '../types/entity.dart';
 import 'plugin.dart';
 
 class Editor {
-  final IosEditor _iOS = const IosEditor();
+  final IosEditor _ios = const IosEditor();
+  final DarwinEditor _darwin = const DarwinEditor();
   final AndroidEditor _android = const AndroidEditor();
 
+  @Deprecated('use [Editor.darwin] instead')
+
+  /// use [Editor.darwin] instead, the field will be removed in 3.0.0
   IosEditor get iOS {
```

Comment:
Still necessary?

Reply:
Here you can directly modify the type of the returned object. For external users, it may be breaking change

Reply:
Because I use `iOS` to inherit `darwin`, not the other way around.